### PR TITLE
Properly respect `numItineraries` config item

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Previously it would always be ignored as core utils always ships a `3`? Curious what I did here, this doesn't feel entirely right

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Comment by daniel-heppner-ibigroup** [comment]
What if we just delete that property in Binh's new function? https://github.com/opentripplanner/otp-react-redux/pull/1226

**Comment by miles-grant-ibigroup** [comment]
> What if we just delete that property in Binh's new function? #1226

But it is used

**Comment by daniel-heppner-ibigroup** [comment]
Are you sure? I think it only gets set in the config, never by a user action

**Comment by miles-grant-ibigroup** [comment]
> Are you sure? I think it only gets set in the config, never by a user action

Field trip sets it upon click

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Field trip strikes again

File: lib/actions/apiV2.js [hunk 1]
```diff
@@ -995,12 +995,12 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
       from: currentQuery.from,
       modes: modes || activeModes,
       modeSettings,
-      numItineraries: numItineraries || config?.modes?.numItineraries || 7,
       time,
       to: currentQuery.to,
       unpreferred,
       // TODO: Does this break everything?
```
Comment (amy-corson-ibigroup) [inline lib/actions/apiV2.js:1001]: Should we delete this one year old TODO

**Review by amy-corson-ibigroup — APPROVED** [review]
Can't find any issues!

File: lib/actions/apiV2.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/actions/apiV2.js:1001]: I'm still not confident

**Pull request merged**